roles/nfs_server_destroy/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+ec2_key: "{{ lookup('ENV', 'EC2_KEY') }}"
```

Comment:
Do we need ec2_key for  destroy? it doesn't seem to be used...

Reply:
No, not at all, could be deleted.